sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2266,6 +2266,7 @@ def __init__(
         self.launched_nodes = launched_nodes
         self.launched_resources = launched_resources
         self.docker_user: Optional[str] = None
+        self.is_grpc_enabled = True
```

Comment:
`skylet_ssh_tunnel = None` could mean two things:
1. The cluster is a legacy cluster, before gRPC was added to skylet
2. The cluster is a brand new cluster, and the SSH tunnel is not yet established.

So we need an additional flag to distinguish between the two.

Reply:
@SeungjinYang this should make sure we can reliably decide when to use gRPC and when to fallback to the legacy execution, without waiting for the 5s gRPC timeout.